ansible/roles/cli_vault/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+- name: Add an Apt signing key for Hashicorp
+  ansible.builtin.apt_key:
+    url: https://apt.releases.hashicorp.com/gpg
+    state: present
+  become: true
+
+# TODO: Find a way to retrieve this information from the system one time
+- name: Found and register "$VERSION_CODENAME"
+  ansible.builtin.shell:
+    cmd: ". /etc/os-release && echo ${VERSION_CODENAME}"
+  become: true
+  register: version_codename
+  changed_when: false
```

Comment:
Have a look at the [ansible facts](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_vars_facts.html#ansible-facts). I think ```{{ ansible_distribution_release }}``` might do the job.

Reply:
I don't understand how it's work :(
We don't have VERSION_CODENAME on the ansible fact ?!